pom.xml

```
@@ -110,4 +129,4 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```

```suggestion
</project>

```
те, за що Богдан говорив